CHANGELOG.md

```diff
@@ -108,6 +108,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - webui and console: add job duration column in job listings [PR #1007]
 - packages: Build EL_8 packages instead of CentOS_8 for RHEL 8 derivates (Alma, Rocky, CentOS Stream)[PR #992]
 - webui: provide a notification that NDMP restores are NOT supported by webui [PR #1020]
+- webui: introduce a job timeline chart (preview) [PR #1017]
```

Comment:
I think we can remove the "(preview)" both from the PR and the description.

Reply:
Removed it.